R/drive_get.R

```diff
@@ -6,7 +6,7 @@
 #'
 #' @description If the files are specified via `path`, the returned [`dribble`]
 #'   will include a `path` variable. To add path information to any [`dribble`]
-#'   that lacks it, use [drive_add_path()]. If you want to list the contents of
+#'   that lacks it, use [drive_reveal()]. If you want to list the contents of
 #'   a folder, use [drive_ls()]. For general searching, use [drive_find()].
```

Comment:
`drive_reveal(what = "path")`?

I will probably dig in a bit here and, if I do, I'll make this change. Or you can and I'll pull.